requirements.txt

```
@@ -1,3 +1,4 @@
-dwave-ocean-sdk==6.4.0
-matplotlib==3.3.4
+dwave-ocean-sdk>=6.4.0
+matplotlib~=3.3.4
```

This limits `matplotlib` to `3.3.x`, i.e. only patch can vary. Quite restricted. 

For matplotlib, it's better to set lower bound only, as it's unlikely to break some basic functionality in the future.
```suggestion
matplotlib>=3.3.4
```